scripts/jenkins/wait-for-cluster.sh

```diff
@@ -0,0 +1,4 @@
+until kubectl cluster-info
```

Comment:
You'll also want a `set -e` here to make the script exit with a non 0 exit code if say `kubectl cluster-info` errors.